Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarCloudWebServiceTest.cs

```diff
@@ -139,7 +149,7 @@ public void GetProperties_Success()
         [TestMethod]
         public void GetProperties_NullProjectKey_Throws()
         {
-            var testSubject = new SonarQubeWebService(new TestDownloader(), uri, version, logger);
+            var testSubject = new SonarCloudWebService(downloader, uri, version, logger, Organization);
```

Comment:
Regarding the name `testSubject`. I would either use the convention `sut` or, if we want to avoid overriding the name, directly `webService` or `server` since it's already used like this in a lot of places.

Reply:
I will use `sut` for consistency.